virt/virtual_machines/advanced_vm_management/virt-vm-control-plane-tuning.adoc

```diff
@@ -0,0 +1,17 @@
+:_content-type: ASSEMBLY
+[id="virt-vm-control-plane-tuning"]
+= Virtual machine control plane tuning
+
+include::_attributes/common-attributes.adoc[]
+:context: virt-control-plane-tuning
+
+OpenShift Virtualization offers several tuning options at the control-plane level. This includes:
+
+* The `highBurst` profile, which uses fixed `QPS` and `burst` rates, to create hundreds of virtual machines (VMs) in one batch.
+* Migration setting adjustment based on workload type.
+
```

Comment:
Where is the migration setting adjustment documented? Is it part of this PR or does it already exist?

Reply:
Documenting VM Migration Tuning was not in this JIRA.

Reply:
So where is this coming from?